src/boi/boi_wifi.cpp

```diff
@@ -105,162 +88,131 @@ void boi_wifi::DisableWiFi()
             yield();
-            LoopCount = 0;
+            LoopCount = 0; //time to reset & go again
         }
         pthread_mutex_unlock(&lock);
         yield();
-        delay(100);
+        delay(100); //read and send sensor data every X milliseconds
     };
-
     pthread_mutex_unlock(&lockDone);
 }
 
-void boi_wifi::monitor_smwn(){
+void boi_wifi::monitor_smwn() { // contains clock cycle for message board pulls
     SensorDataStruct SensorData;
     int LoopCount;
     int LoopScanCount;
     uint64_t NextGetCheck;
-
     pthread_mutex_lock(&lockDone);
-
     LoopCount = 0;
     LoopScanCount = 0;
     NextGetCheck = esp_timer_get_time() + (10*1000*1000);
-
-    while(1)
-    {
-        if(pthread_mutex_trylock(&lock))
-        {
+    while(1) {
+        if(pthread_mutex_trylock(&lock)) {
             Serial.println("Exiting thread");
             break;
         }
-
-        //increment our loop count
-        LoopCount++;
+        LoopCount++; //increment our loop count
         LoopScanCount++;
-        if(LoopCount >= 5)
-        {
-            //read and send sensor data roughly every half a second
-            //not exact due to timing and other activities on the device but this does not have
-            //to be accurate
+        if(LoopCount >= 5) { //causes delay in this while to not be exact due to timing and other activities on the device, but this does not have to be accurate
             this->_boi->get_sensor_data(&SensorData);
             yield();
             this->message_handler->HandleSensorData(&SensorData);
             yield();
             LoopCount = 0;
         }
-
-        if(esp_timer_get_time() >= NextGetCheck) {
-            //process messages, if we had messages then check every 10 seconds otherwise every minute
-            if(this->message_handler->QueryBatteryInternet())
+        if(esp_timer_get_time() >= NextGetCheck) { //process messages, if we had messages then check every 10 seconds otherwise every minute
+            if(this->message_handler->QueryBatteryInternet()) {
                 NextGetCheck = esp_timer_get_time() + (10*1000*1000);
-            else
+            }
+            else {
                 NextGetCheck = esp_timer_get_time() + (60*1000*1000);
+            }
         }
-
         pthread_mutex_unlock(&lock);
-
         yield();
-        delay(100);
+        delay(100); //read and send sensor data every X milliseconds
     };
-
     pthread_mutex_unlock(&lockDone);
 }
 
-void boi_wifi::Reconfigure(const OptionsStruct *Options)
-{
+void boi_wifi::Reconfigure(const OptionsStruct *Options) {
     char WifiName[20 + 1 + 10];
     // uint8_t BatterySymbol[] = " \xF0\x9F\x94\x8B "; // Standard edition, but who wants that?
     // uint8_t BatterySymbol[] = " \xF0\x9F\x8E\x83 "; // SPOOKY EDITION OOOOOOOOOoOOOoOooooooooo.............
     uint8_t BatterySymbol[] = " \xF0\x9F\x8E\x83 \xF0\x9F\x98\xB7 \xF0\x9F\x8E\x83 "; // SAFE MODE WITH NETWORKING EDITION - WASH YOUR HANDS!!!!!
```

Comment:
We should pick a new Emoji to broadcast with the captive portal.

Reply:
The theme of DEF CON this year is "The Signal" to connect with one another. I like the idea of an antenna emoji or something else that indicates connecting. 

📡 or something like it.